app/src/main/java/com/example/natallia_radaman/simplerecyclerview/ArticlesRecyclerAdapter.java

```diff
@@ -0,0 +1,54 @@
+package com.example.natallia_radaman.simplerecyclerview;
+
+import android.content.Context;
+import android.support.annotation.NonNull;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.example.natallia_radaman.simplerecyclerview.HardCodeContent.HardCodeArticle;
+
+import java.util.List;
+
+public class ArticlesRecyclerAdapter extends RecyclerView.Adapter<ArticlesViewHolder> {
+
+    private List<HardCodeArticle> articleList;
+    private Context context;
+    private FragmentCoordinator callbackActivity;
+
+    public ArticlesRecyclerAdapter(List<HardCodeArticle> articleList, Context context) {
```

Comment:
Will articleList or context change later in constructor body? If not, then why not add `final` modifier?